### PR TITLE
Feature/organize scss files

### LINKED ISSUE #12: Current Site is Hardcoded for Activity data
**As a user:**
* I would like to see relevant data from `Activity.js`

### LINKED ISSUE #18: Separate file for mixins
As a dev

If converting css to scss I would like to have a separate file for mixins 

To better organize the data in the css folder

### PR DESCRIPTION
**Pull request by okalimbet**
## What is the change?
Covrted css style to scss by creating mixins, color functions, and nesting html elements.

## What does it fix? 
Scss styling

## Is this a fix or a feature?  
Fix css style and converting it to scss

## Where should the reviewer start?
mixins.scss:
line 1

style.scss:
line 2

variables.scss:
line 4-7 - new color functions
line 25-39, 170-174, 176-188 - fixing indentations

## How should this be tested?
By running npm start and opening the website the website displays all necessary information and styles.

### CODE REVIEW

File: src/css/mixins.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
+@import "variables";
+
+@mixin flex-row {
```
Comment (mdflynn) [inline src/css/mixins.scss:3]: Nice use of the mixin!

File: src/css/mixins.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@import "variables";
```
Comment (cameronRomo) [inline src/css/mixins.scss:1]: I love this new use of mixins. They really helped to dry up our scss file.

**Review by mdflynn — APPROVED** [review]
The mixins are a nice addition to the application!

File: src/css/style.scss [hunk 3]
```diff
@@ -1,4 +1,5 @@
 @import "variables";
+@import "mixins";
```
Comment (cameronRomo) [inline src/css/style.scss:2]: Nice use of mixins and nesting here. Really cleans up our scss

File: src/css/variables.scss [hunk 4]
```diff
@@ -1,14 +1,15 @@
 $colorPrimary: #4d4e4f;
 $colorHeading: #ffffff;
 $colorBorder: black;
-$colorHydration: #5BC8AC;
-$colorSleep: #EC96A4;
-$colorActivity: #DFE166;
+$colorHydration: mix(white, #00AA00, 40%);
```
Comment (cameronRomo) [inline src/css/variables.scss:4]: Color functions are interesting. I like the effect they've already had on our site.

**Review by cameronRomo — APPROVED** [review]
Very nice refactor of the styles. It's a way cleaner way to write the files.

**Pull request merged**